### ISSUE TITLE
Can't unlink dojo

### ISSUE THREAD

**Issue by juped** [urbit/urbit#452]
There's no way to unlink a stuck dojo short of editing a crash into ape/dojo

**Comment by dhaffey** [comment]
My low-tech solution to this is an `|unlink-dojo` hood generator. It's just a copy-pasted version of `|unlink` with a hardcoded `%dojo` argument, but that's enough to avoid the need for dojo to evaluate anything before sending the poke to hood.

**Comment by juped** [comment]
No, because if your dojo works, you can already |unlink it just fine.

**Comment by dhaffey** [comment]
The whole point of |unlink-dojo is that it works when |unlink doesn't. At least in the situations I needed it, maybe we're talking about different sorts of stuck dojo.

**Comment by juped** [comment]
The only situation where it doesn't is when you can't type into dojo - `|unlink %dojo` works. If you have other dojo bugs, please report them!

**Comment by dhaffey** [comment]
Now that I remember what I was using it for, it wasn't actually necessary, but I was pretty confused by the state I landed in the first time it happened. I had done something like push a library with /+ and then change to a desk without that file. In that state (or any time something in =lib won't compile), dojo bombs out on the bad lib when evaluating expressions, including arguments to %= and generators. Later I realized =lib would do the trick, that's all |unlink-dojo was really doing for me.
